### DISCUSSION TITLE
Future of Flux Extension after Weaveworks dropping

### DISCUSSION THREAD

**Discussion by wsmelton** [Azure/AKS#4120] [General]
A bit linked to this discussion [Is the project future at risk](https://github.com/fluxcd/flux2/discussions/4544) from Flux repo.

I'm curious if Microsoft team is going to change any stance on that now with maintaining the images and security of them. If you note one company has already stepped in to offer secure images for the controllers but have put that behind a pay-wall.

**Comment by philwelz** [comment]
Hi, as [announced](https://www.cncf.io/announcements/2024/03/19/cloud-native-computing-foundations-fluxcd-project-gains-new-corporate-support/) during Kubecon Microsoft and several other companies will support the Flux Project.  So I think Flux will be in a better shape soon as more companies will contribute.
